playbooks/group_vars/load-balancer/public.yml

```diff
@@ -17,8 +17,8 @@ SANITY_CHECK_LIVE_PORTS:
     message: "Load balancer does not respond on port 443."
 
 # TARSNAP #####################################################################
-
-TARSNAP_BACKUP_FOLDERS: "/etc/letsencrypt /etc/haproxy {{ consul_data_dir }}"
+  
+TARSNAP_BACKUP_FOLDERS: "/etc/letsencrypt /etc/haproxy {{ consul_data_dir }} /var/lib/nomad"
```

Comment:
Note that this is a global variable for all load balancers, including ones not running Nomad masters like the current HAProxy prod/stage. So to prevent any errors without having to make a whole new group vars file and change up the SSH config a lot, I just went in and `mkdir /var/lib/nomad` on those servers.

Reply:
You could conditionally add the directory to the list, based on whether the host is in the "nomad" group, e.g.

    TARSNAP_BACKUP_FOLDERS: "/etc/letsencrypt /etc/haproxy {{ consul_data_dir }} {{ '/var/lib/nomad' if 'nomad' in group_names else '' }}"

(I did not test this, but something along this lines should work.)

Reply:
Yep cool, that works!